tests/prompts/test_stream.py

```diff
@@ -28,14 +28,15 @@ def gpt4_behaviour():
     return behaviour.AIModelsBehaviour(
         attempts=[
             AttemptToCall(
-                ai_model=OpenAIModel(
-                    model="gpt-4o",
+                ai_model=AzureAIModel(
+                    realm='westus',
+                    deployment_id="gpt-4-turbo",
                     max_tokens=C_128K,
                     support_functions=True,
-                    should_verify_client_has_creds=False,
+                    should_verify_client_has_creds=False
                 ),
                 weight=100,
-            ),
+            )
         ]
     )
     
```

Comment:
can you add tests? Please add in template of PR a question if a person added a mechanism, if it was bug-fix